fhir-persistence-jdbc/pom.xml

```diff
@@ -120,6 +120,11 @@
             <artifactId>jsonassert</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.enterprise.concurrent</groupId>
+            <artifactId>javax.enterprise.concurrent-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
is this actually used? or did you pull that out when you simplified things?

Reply:
No - pulled out. great catch.